doc/user/rbac.md

```diff
@@ -1,27 +1,27 @@
-# Operator RBAC setup
+# Setting up RBAC for etcd-operator
```

Comment:
etcd-operator -> etcd operator

See discussion in https://github.com/coreos/etcd-operator/pull/1709/files#r154206953